src/UserBundle/Maker/UserMaker.php

```diff
@@ -157,12 +157,12 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                 } else {
                     $io->writeln($differ->diff($exist ? file_get_contents($file) : '', $contents));
                 }
-            } while ('r' === $choice = $io->choice('Write changes and continue reviewing?', $choices, $choices['r']));
+            } while ( !$input->getOption('no-interaction') || ('r' === $choice = $io->choice('Write changes and continue reviewing?', $choices, $choices['r'])) );
```

Comment:
i suggest to rewrite `$review` above like

`$review = $io->confirm('...', $input->isInteractive());`